From 8711db616a17523abcea9615c56233c68cf6a1e5 Mon Sep 17 00:00:00 2001
From: Ben Cressey <bcressey@amazon.com>
Date: Sun, 15 Sep 2019 00:51:25 +0000
Subject: [PATCH 9002/9005] do not create unused state directories

We do not use the coredump handler, and the private directories have
been relocated to `/run`.

Signed-off-by: Ben Cressey <bcressey@amazon.com>
---
 tmpfiles.d/systemd.conf.m4 | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/tmpfiles.d/systemd.conf.m4 b/tmpfiles.d/systemd.conf.m4
index 11d87d2..c8fb51a 100644
--- a/tmpfiles.d/systemd.conf.m4
+++ b/tmpfiles.d/systemd.conf.m4
@@ -65,10 +65,3 @@ a+ /var/log/journal    - - - - d:group::r-x,d:group:wheel:r-x,group::r-x,group:w
 a+ /var/log/journal/%m - - - - d:group:wheel:r-x,group:wheel:r-x
 a+ /var/log/journal/%m/system.journal - - - - group:wheel:r--
 '')')')m4_dnl
-
-d /var/lib/systemd 0755 root root -
-d /var/lib/systemd/coredump 0755 root root 3d
-
-d /var/lib/private 0700 root root -
-d /var/log/private 0700 root root -
-d /var/cache/private 0700 root root -
-- 
2.21.0

